dotnet6/cookiecutter-aws-sam-quickstart-sns-dotnet/{{cookiecutter.project_name}}/test/project.Tests/FunctionTest.cs

```diff
@@ -0,0 +1,37 @@
+using Xunit;
```

Comment:
@neville-lewis please rename the directory back to {{cookiecutter.project_name}} instead of project.Tests